builder/gen-dockerfile/contents/src/detect-setup.ts

```diff
@@ -0,0 +1,134 @@
+/*
```

Comment:
I can remove the fingerprinting code.  The part of the detection that will be needed is `yarn`/`npm` detection and other detection that is used to generate the `Dockerfile`.